Singer.API/ClientApp/angular.json

```diff
@@ -22,7 +22,7 @@
                   "tsConfig": "src/tsconfig.app.json",
                   "assets": ["src/assets"],
                   "styles": [
-                     "src/styles.css"
```

Comment:
Since when are we using SASS?

Reply:
Since we need to leverage the angular material theme mixin methods

Reply:
It's only for our main styles.scss the global project setting still uses css.

Reply:
If i leave it as a .css file the production build will complain that it does not recognize the $... scss variables in the custom theme.